lms/djangoapps/grades/models.py

```diff
@@ -336,3 +340,59 @@ def update(
         self.possible_graded = possible_graded
         self.visible_blocks_id = visible_blocks_hash  # pylint: disable=attribute-defined-outside-init
         self.save()
+
+
+class SubsectionGradesEnabledFlag(ConfigurationModel):
```

Comment:
Let's try to include "Persistent" in the model name somewhere, as subsections are still graded regardless of this setting.